src/convert.rs

```diff
@@ -814,12 +814,39 @@ pub fn elf_to_tbf(
             }
         }
 
-        // Save the end of this segment so we can check if padding is required
-        // between segments.
-        last_segment_address_end = Some(end_segment as usize);
+        if reloc_only {
+            if verbose {
+                println!("    Skipping segment since it only contains relocs");
+            }
```

Comment:
Can we just put a `continue` here (consistent with the checks above around line 629?